scapy/layers/hsrp.py

```diff
@@ -67,7 +67,7 @@ class HSRPmd5(Packet):
         ByteEnumField("algo", 0, {1: "MD5"}),
         ByteField("padding", 0x00),
         XShortField("flags", 0x00),
-        IPField("sourceip", None),
+        IPField("sourceip", "127.0.0.1"),
```

Comment:
Now if you rebase against master, you can change this to `SourceIPField("sourceip", None)`